redis/src/connection.rs

```diff
@@ -760,6 +760,23 @@ pub fn connect(
     setup_connection(con, &connection_info.redis)
 }
 
+pub(crate) fn client_set_info_pipeline() -> Pipeline {
+    let mut pipeline = crate::pipe();
+    pipeline
+        .cmd("CLIENT")
+        .arg("SETINFO")
+        .arg("LIB-NAME")
+        .arg(std::env!("BABUSHKA_NAME"))
+        .ignore();
```

Comment:
What happens in redis < 7? does it fail in silent?

Reply:
also - I think we should add a comment stating that it works only for >= V7

Reply:
> What happens in redis < 7? does it fail in silent?
yes. I'll add a comment.

Reply:
it's supposed to fail silently, and there's a comment saying that.